pages/common/micro.md

```diff
@@ -9,20 +9,20 @@
 
 - Cut the entire line:
 
-`<Ctrl>k`
+`<Ctrl> + K`
 
 - Search for a pattern in the file (press `<Ctrl>n`/`<Ctrl>p` to go to next/previous match):
```

Comment:
Could you change the shortcuts here? And please could you also remove the `<` and `>` around the `Ctrl` for each shortcut.

Other than that, all looks fine to me.